app/views/ExposureHistory/__tests__/ExposureHistory.spec.js

```diff
@@ -23,7 +23,9 @@ describe('convertToDailyMinutesExposed', () => {
   });
 
   it('converts day bins to minutes', () => {
-    expect(convertToDailyMinutesExposed('[0, 1, 2, 3, 0]')).toEqual([
+    expect(
```

Comment:
check non even spaced minutes to see what we get

Reply:
I'm going to leave this alone for now ... a little beyond the scope of what this is trying to fix.